### PR TITLE
MINOR: minor fixes

### PR DESCRIPTION
**Pull request by guozhangwang**
0. Bump up the AK version to 0.11.1.0 (will need to create 3.3.x branch that depends on 0.11.0.0 after this is merged).

1. Set the log4j default entry to INFO for users to be able to check the log4j outputs, also update the corresponding README file to indicate console producer / consumer are usually needed.

2. Specify the client-id in all the examples to make the thread-id be more readable.

### CODE REVIEW

File: kafka-streams/README.md [hunk 1]
```diff
@@ -343,12 +343,30 @@ $ java -cp target/streams-examples-3.3.0-SNAPSHOT-standalone.jar \
   io.confluent.examples.streams.WordCountLambdaExample
 ```
 
+which will then try to read from the specified input topic (in the above example it is ``TextLinesTopic``),
```
Comment (mjsax) [inline kafka-streams/README.md]: `The application will try to read`

File: kafka-streams/README.md [hunk 2]
```diff
@@ -343,12 +343,30 @@ $ java -cp target/streams-examples-3.3.0-SNAPSHOT-standalone.jar \
   io.confluent.examples.streams.WordCountLambdaExample
 ```
 
+which will then try to read from the specified input topic (in the above example it is ``TextLinesTopic``), 
+execute the processing logic, and finally try to write back to the specified output topic (in the above example it is ``WordsWithCountsTopic``).
+So in order to observe the expected output stream, you will need to also start a console producer to send messages into the input topic
```
Comment (mjsax) [inline kafka-streams/README.md]: Remove `So` -> `In order to observe`
Remove `also` -> `need to start a`

File: kafka-streams/README.md [hunk 3]
```diff
@@ -343,12 +343,30 @@ $ java -cp target/streams-examples-3.3.0-SNAPSHOT-standalone.jar \
...
+and start a console consumer to continuously read  from the output topic. More details in how to run the examples can be found
```
Comment (mjsax) [inline kafka-streams/README.md]: Nit: double space `read from`

File: kafka-streams/README.md [hunk 4]
```diff
@@ -343,12 +343,30 @@ $ java -cp target/streams-examples-3.3.0-SNAPSHOT-standalone.jar \
...
+in the java docs of each example code.
```
Comment (mjsax) [inline kafka-streams/README.md]: `JavaDocs` (maybe add link?)

File: kafka-streams/src/main/java/io/confluent/examples/streams/ApplicationResetExample.java [hunk 5]
```diff
@@ -115,6 +115,7 @@ public static void main(final String[] args) throws Exception {
     // Give the Streams application a unique name.  The name must be unique in the Kafka cluster
     // against which the application is run.
     streamsConfiguration.put(StreamsConfig.APPLICATION_ID_CONFIG, "application-reset-demo");
+    streamsConfiguration.put(StreamsConfig.CLIENT_ID_CONFIG, "interactive-queries-example-client");
```
Comment (miguno) [inline kafka-streams/src/main/java/io/confluent/examples/streams/ApplicationResetExample.java]: The client id (IQ) doesn't match the app id (app reset).

Comment (guozhangwang) [inline kafka-streams/src/main/java/io/confluent/examples/streams/ApplicationResetExample.java]: Ack.

**Comment by guozhangwang** [comment]
@mjsax @miguno comments addressed, thanks!

**Pull request merged**